### PR TITLE
Feature/add metrics

### PR DESCRIPTION
**Pull request by jedi7**
Extending support for standard units on m-bus

### CODE REVIEW

**Comment by jedi7** [comment]
please just give me known how big the BITBUF_COLS can be, and I will update it.

**Comment by jedi7** [comment]
I have prepared also branch which add timedate, and slightly modify the append function:
https://github.com/jedi7/rtl_433/pull/new/feature/add_mbus_datetime

File: include/bitbuffer.h [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 #include <stdint.h>
 
-#define BITBUF_COLS 256 // Number of bytes in a column
+#define BITBUF_COLS 512 // Number of bytes in a column
```
Comment (zuckschwerdt) [inline include/bitbuffer.h:17]: Make it 450 then and write a comment that m_bus needs this.

File: src/devices/m_bus.c [hunk 2]
```diff
@@ -210,30 +196,265 @@ static char* oms_hum_el[4][4] = {
 {"Error 31","Error 32","Error 33","Error 34",}
 };
 
+static char* history_hours[4] = {
```
Comment (zuckschwerdt) [inline src/devices/m_bus.c:199]: type should be `char *x` instead of `char* x`.
indent these lists by 8 spaces.

Comment (jedi7) [reply src/devices/m_bus.c:199]: @zuckschwerdt  are you sure? I see the "static char* oms_hum" uses c++ asterix position (like type is char*, not *osm_hum). Indentation the same. I can change it every where to have it consistent. If needed :)

File: include/bitbuffer.h (hunk 1 above)
Comment (jedi7) [reply include/bitbuffer.h:17]: fixed

File: src/devices/m_bus.c (hunk 2 above)
Comment (zuckschwerdt) [reply src/devices/m_bus.c:199]: Yes, but only fix your code for now to keep the diff readable. You can also check with clang-format for a rough guide.

**Pull request merged**